s3/client.bal

```diff
@@ -236,38 +249,38 @@ public isolated client class Client {
         if (versionId is string) {
             queryParamsStr = string `${queryParamsStr}?versionId=${versionId}`;
             queryParamsMap["versionId"] = versionId;
-        }    
+        }
         map<string> requestHeaders = setDefaultHeaders(self.amazonHost);
-        
+
         check generateSignature(self.accessKeyId, self.secretAccessKey, self.region, DELETE, requestURI,
-            UNSIGNED_PAYLOAD, requestHeaders, request, queryParams = queryParamsMap);
+                UNSIGNED_PAYLOAD, requestHeaders, request, queryParams = queryParamsMap, sessionToken = self.sessionToken);
         requestURI = string `${requestURI}${queryParamsStr}`;
         http:Response httpResponse = check self.amazonS3->delete(requestURI, request);
         return handleHttpResponse(httpResponse);
-    }     
+    }
 
     # Deletes a bucket.
-    # 
+    #
     # + bucketName - The name of the bucket
     # + return - An error on failure or else `()`
     @display {label: "Delete Bucket"}
     remote isolated function deleteBucket(@display {label: "Bucket Name"} string bucketName) returns @tainted error? {
         http:Request request = new;
         string requestURI = string `/${bucketName}`;
         map<string> requestHeaders = setDefaultHeaders(self.amazonHost);
-        
+
         check generateSignature(self.accessKeyId, self.secretAccessKey, self.region, DELETE, requestURI,
-            UNSIGNED_PAYLOAD, requestHeaders, request);
+                UNSIGNED_PAYLOAD, requestHeaders, request, sessionToken = self.sessionToken);
         http:Response httpResponse = check self.amazonS3->delete(requestURI, request);
         return handleHttpResponse(httpResponse);
     }
-    
+
     # Generates a presigned URL for the object.
     #
     # + bucketName - The name of the bucket  
     # + objectName - The name of the object  
     # + action - The action to be done on the object (`RETRIEVE` for object retrieval or `CREATE` for object creation) 
-    #            or the relevant headers for object retrieval or creation
+    # or the relevant headers for object retrieval or creation
```

Comment:
If possible, shall we plan to introduce all the formatting fixes as a separate PR(maybe later)? That will make it lot more easier to see the actual functionality changes introduced by this change IMO